pkg/brokerapi/openservicebroker/exponential_backoff_poller.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openservicebroker
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	"github.com/kubernetes-incubator/service-catalog/pkg/brokerapi"
+)
+
+const (
+	// Max specifies the maximum duration we're going to back off up to
+	max = 1 * time.Hour
```

Comment:
:) I have this hazy recollection that some brokers can take hours. But, we can tweak this as necessary... Thanks for the datapoint!